frontend/resources/translations/de-DE.json

```diff
@@ -741,17 +846,21 @@
         "left": "Links",
```

Comment:
Zeile 768: "selectADocumentToLinkTo": "Eine Tabelle zum Verknüpfen auswählen"
Zeile 777: "referencedGrid": "Referenzierte Tabelle"
Zeiel 825: "imageSavingFailed": "Das Bild konnte nicht gespeichert werden"
Zeile 841: "turnInto": "Einfügen in"